paddle/framework/lod_tensor.h

```diff
@@ -188,5 +188,14 @@ std::pair<LoD, std::pair<size_t, size_t>> GetSubLoDAndAbsoluteOffset(
 
 void AppendLoD(LoD* lod, const LoD& lod_length);
 
+/*
+ * Serialize/Desiralize LoDTensor to std::ostream
+ * You can pass ofstream or ostringstream to serilize to file
+ * or to a in memory string. GPU tensor will be copied to CPU.
+ */
+void SerializeToStream(std::ostream& os, const LoDTensor& tensor,
```

Comment:
For discussion: consider make `SerializeToStream` and `DeserializeFromStream` member functions for `LoDTensor`. Reason stated here: https://github.com/PaddlePaddle/Paddle/pull/5520/files#r151577859
CC: @dzhwinter